src/components/HomePage/SiteMask.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './styles/SiteMask.css';
+
+let cx = classNames.bind(styles);
+
+class SiteMask extends React.Component {
```

Comment:
Do we need this new component? Is it possible that we could take the functionality here and merge it into Background.jsx component? That way we can have one sole component that handles anything to do with the background. We could also even move this class into the Background.jsx file and not export it.